test/unit/frontend/delete-github-subscription.test.ts

```diff
@@ -1,8 +1,19 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { Installation, Subscription } from "../../../src/models";
 import { GithubSubscriptionDelete } from "../../../src/routes/github/subscription/github-subscription-delete";
+import { when } from "jest-when";
+import { booleanFlag, BooleanFlags } from "../../../src/config/feature-flags";
+
+jest.mock("../../../src/config/feature-flags");
+
+const createGitHubNockGet = (url, status, response) => {
+	githubNock
+		.get(url)
+		.reply(status, response);
+};
+
+describe("POST /github/subscription - octokit", () => {
```

Comment:
Split Octokit out into its own test, since there was only 1 test using Octokit and it wasnt using Nock, was easier to have as a seperate describe block, have moved and updated existing tests inside the main describe.